scripts/run-prepare.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+TAG="[widget-driver]:"
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+TAG_COLOR="\n${GREEN}${TAG}${NC}"
+PROJECT_FOLDERS=("widget_driver" "widget_driver_annotation" "widget_driver_generator" "widget_driver_test")
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function run_prepare() {
+    flutter pub get 
+}
+
+function fetch_pods() {
+    current_dir=$(pwd)
+    echo -e "$TAG_COLOR Running pod install"
+    cd "widget_driver/example/ios"
+    pod repo update
+    pod install
+    cd $current_dir
+}
```

Comment:
can we make it optional to run any iOS specific stuff? We would want to run the pipeline on linux nodes anyways, therefore I see no need to fetch iOS dependencies.